CONTRIBUTING.md

```diff
@@ -142,20 +142,20 @@ tracing) enabled.
 The setup for a local rustc works as follows:
 ```sh
 # Clone the rust-lang/rust repo.
-git clone https://github.com/rust-lang/rust/ rustc
+git clone https://github.com/rust-lang/rust rustc
 cd rustc
 cp config.toml.example config.toml
 # Now edit `config.toml` and set `debug-assertions = true`.
 
-# Build a stage 2 rustc.
+# Build a stage 1 rustc.
 # This step can take 30 minutes or more.
-./x.py build --stage 2 compiler/rustc
+./x.py build --stage 1 compiler/rustc
 # If you change something, you can get a faster rebuild by doing
-./x.py build --keep-stage 0 --stage 2 compiler/rustc
+./x.py build --keep-stage 0 --stage 1 compiler/rustc
```

Comment:
This should no longer be necessary; if you change something in rustc, you do actually have to rebuild to get the right behavior.
```suggestion
```

Reply:
Err wait, `compiler/rustc` is still building the compiler twice. So I guess this does need to be kept :/ it would be good to figure out why `compiler/rustc` is necessary.

Reply:
> it would be good to figure out why compiler/rustc is necessary.

As I keep saying, this is expected since we are linking against the compiler and using proc-macros and not using any magic like bootstrap does to use a different rustc for the proc-macros and the actual crate being built.